pkgs/by-name/py/pychess/package.nix

```
@@ -0,0 +1,74 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  pkg-config,
+  gobject-introspection,
+  wrapGAppsHook3,
+  gtk3,
+  gst_all_1,
+  gtksourceview,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pychess";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "pychess";
+    repo = "pychess";
+    rev = "${version}";
+    sha256 = "sha256-hxc+vYvCeiM0+oOu1peI9qkZg5PeIsDMCiydJQAuzOk=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    gobject-introspection
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    gtk3
+    gst_all_1.gst-plugins-base
+    gtksourceview
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    pygobject3
+    pycairo
+    sqlalchemy
+    pexpect
+    psutil
+    websockets
+    ptyprocess
+  ];
+
+  dontWrapGApps = true;
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  preBuild = ''
+    export HOME=$(mktemp -d)
+    export PYTHONPATH=./lib:$PYTHONPATH
+    ${python3Packages.python.interpreter} pgn2ecodb.py
+    ${python3Packages.python.interpreter} create_theme_preview.py
```

```suggestion
    python pgn2ecodb.py
    python create_theme_preview.py
```

You won't want that `buildPackages...` stuff I suppose.